File: Python Word Examples/ConvertApiDemo/convert.py
import spirecloudword
from spirecloudword import *

appId = "your id"
appKey = "your key"
baseUrl = "https://api.e-iceblue.cn"
configuration = Configuration(appId, appKey, baseUrl)
api = spirecloudword.api.convert_api.ConvertApi(configuration)

file_name = "convert.docx"
target_format = 'postScript' #Available values: doc, docx, wordml, wordxml, odt, docm, dotm, dotx, dot, rtf, pdf, xps, epub, pcl, postScript
dest_file_path = 'output/convert.ps'
extra = {'folder': 'input'}
api.convert(file_name, target_format, dest_file_path, **extra)

File: PHP Word Examples/ConvertApiDemo/convert.php
<?php

use Spire\Cloud\Word\Sdk\Client\Api\ConvertApi;
use Spire\Cloud\Word\Sdk\Client\Configuration;

$appId = "your id";
$appKey = "your key";
$baseUrl="https://api.e-iceblue.cn";
$configuration = new Configuration($appId, $appKey,$baseUrl);
$apiInstance = new ConvertApi($configuration);

$name = "convert.docx";
$folder = "input";
$storage = null;
$format = 'postScript'; //Available format: pdf, pcl, dot, dotx, dotm, docm, odt, wordxml, wordml, xps, docx, doc, rtf, epub, xps, html
$password = null;
$dest_file_path = "output/convert.ps";
$apiInstance->convert($name, $format,$dest_file_path, $password, $folder, $storage);